templates/infrastructure/account/tfEnvironment/tfRegion/mongo_atlas/terragrunt.hcl

```diff
@@ -0,0 +1,50 @@
+include {
+  path = find_in_parent_folders()
+}
+
+locals {
+  # Automatically load environment-level variables
+  environment_vars = read_terragrunt_config(find_in_parent_folders("env.hcl"))
+  region_vars = read_terragrunt_config(find_in_parent_folders("region.hcl"))
+
+  env = local.environment_vars.locals.environment
+  region = local.region_vars.locals.aws_region
+}
+
+# Terragrunt will copy the Terraform configurations specified by the source parameter, along with any files in the
+# working directory, into a temporary folder, and execute your Terraform commands in that folder.
+terraform {
+  # the below config is an example of what the config should like
+  source = "github.com/argonautdev/tf-modules.git//modules/aws/mongo_atlas?ref={{.RefVersion}}"
```

Comment:
Your folder name must be same as the field-name in `TerragruntParser` structs. Change this folder name to `mongo_atlas_vpc_peering` or change name in midgard to `mongo_atlas`.

Reply:
Changing it to `mongo_atlas_vpc_peering` is better since you won't need to update the `parser` binary (rebuild midgard-tf image).

Reply:
Done.